### ISSUE TITLE
login-stats not working anymore

### ISSUE THREAD

**Issue by ineiti** [c4dt/stats#5]
Since the update to the new @dedis/cothority npm, login-stats are not working anymore. When running manually, the following happens:

```
./omniledger.login-matrix 
error: failed to select elements from selection 'Button: "Login" [single]': element not found
stats@switch:~$ ./omniledger.login-wordpress 
error: failed to select elements from selection 'Button: "Login" [single]': element not found
stats@switch:~$ 
```

**Comment by tharvik** [comment]
It seems that the whole login is currently broken, I'm unable to login myself.

For more informative message in the CAS test, see https://github.com/c4dt/omniledger/pull/257.

**Comment by ineiti** [comment]
What is not working? I just logged in both to matrix and c4dt, and it seems to work. Although I have an additional `I trust this address` when doing a login to matrix.

**Comment by tharvik** [comment]
I'm unable to do so, it fails with my user-data, and the ones used by the stats' scripts.
It shows "Something went wrong while loading your user" and the logs show some connection errors to some servers (naming conode.c4dt.org:777*) then finishes with `(Error: unknown reason,Error: unknown reason)`.

**Comment by ineiti** [comment]
Should be fixed by c4dt/omniledger#258

**Comment by tharvik** [comment]
Indeed, it's working now.
